monai/apps/pathology/losses/hovernet_loss.py

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+import torch
+from torch.nn import CrossEntropyLoss
+from torch.nn import functional as F
+from torch.nn.modules.loss import _Loss
+
+from monai.losses import DiceLoss
+from monai.transforms import SobelGradients
+from monai.utils.enums import HoVerNetBranch
+
+
+class HoVerNetLoss(_Loss):
+    """
+    Loss function for HoVerNet pipeline, which is combination of losses across three branches
+
+    Args:
```

Comment:
Hi @drbeh @JHancox ,

Could you please complete the doc-string for args?

Thanks in advance.